trite-osprey/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
mocha docs suggest not using arrow functions. `it('renders without crashing', function(){`